### ISSUE TITLE
Uptime, status and performance badges

### ISSUE THREAD

**Issue by MickaelBergem** [HowFast/roadmap#5]
Your server has 100% uptime? Answers blazing fast? Show it off to your clients, collaborators and team members with these little badges.

For now, uptime badges are available in PNG or SVG format: ![uptime](https://badge.howfast.tech/uptime.png?t=https%3A%2F%2Fwww.howfast.tech%2F)

Other badges have been designed but have not yet been implemented due to lack of demand:
![image](https://user-images.githubusercontent.com/3510190/45313180-6ad0ec00-b52e-11e8-8427-dd365b74d711.png)

Let me know if you like this feature and which badge you would adopt in the comments!

**Comment by jpcaruana** [comment]
- uptime in %
- number of days up
- number of days up within the SLA

I would not use the detailed down badges I think

**Comment by MickaelBergem** [comment]
@jpcaruana how would you define the SLA? Can you provide an example?

**Comment by jpcaruana** [comment]
@MickaelBergem sorry for the looooong delay

I would say something like `x% of success requests under x ms` ? also `y% of successful request overall` would be interresting

what do you think ?

**Comment by MickaelBergem** [comment]
@jpcaruana thanks for providing the details!

Regarding the second metric "Y% of successful requests", isn't it already was is defined as "uptime" in HowFast? Or maybe you want to define a budget and be alerted when uptime goes below a given threshold?

**Comment by jpcaruana** [comment]
> Regarding the second metric "Y% of successful requests", isn't it already was is defined as "uptime" in HowFast?

yes, I just wanted to list the metrics I need and I included the existing ones :)
